sabnzbd/newsunpack.py

```diff
@@ -534,7 +534,7 @@ def rar_extract_core(rarfile_path, numrars, one_folder, nzo, setname, extraction
 
```

Comment:
This change is later removed because we remove the check for Expected files, so not important anymore.